### PR TITLE
NGS installer should install NGS-compatible NSC

### PR DESCRIPTION
**Pull request by philpennock**

### CODE REVIEW

**Review by kozlovic — COMMENTED** [review]
Sorry, but I have no idea, deferring to Alberto and Matthias.

**Review by wallyqs — APPROVED** [review]
LGTM

File: install.py [hunk 1]
```diff
@@ -43,7 +43,8 @@
 
 
 NSC_REPO_URL = "https://github.com/nats-io/nsc"
-NSC_LATEST_RELEASE_URL = NSC_REPO_URL + "/releases/latest"
+# NSC_LATEST_RELEASE_URL = NSC_REPO_URL + "/releases/latest"
+NSC_PROD_RELEASE_URL = NSC_REPO_URL + "/releases/tag/0.5.0"
```
Comment (wallyqs) [inline install.py:47]: think it could be v1.0.0 as well

Comment (wallyqs) [reply install.py:47]: ah no it has to be v0.5.0 because there are no artifacts for the v1.0.0 tag :/ 
https://github.com/nats-io/nsc/releases/tag/1.0.0

**Pull request merged**